### PR TITLE
add fhir-sandbox configs for lungair

### LINKED ISSUE #9: add instructions for setting up local fhir server
We will use https://github.com/KitwareMedical/fhir-sandbox

We want to create a custom dataset that can read from our research data tables or generate similar fake data for the sake of a fully open source demo.

~~This depends on #6 being completed~~ (Nevermind; this is just about setting up a local fhir server, not necessarily also talking to it from the web app.)

### PR DESCRIPTION
**Pull request by ebrahimebrahim**
Closes #9

- [x] add a fhir sandbox config that uses our EHR data from the LungAIR research collaboration
  - [x] heart rate
  - [x] fio2
  - [x] sao2
  - [x] pip
  - [x] peep
  - [x] respiratory rate
- [x] document in readme how to set up this sandbox fhir server if you have the EHR data table
- [x] add an alternative synthetic data mode for when a user doesn't have the EHR data table, since the data is not public but this open source project should be fully demonstratable
- [x] document in readme how to use the synthetic data mode

### CODE REVIEW

**Comment by ebrahimebrahim** [comment]
@jadh4v When reviewing this there is no need to look too carefully at the stuff in `fhir-sandbox-config` as it's largely independent from the rest of the application. The main area where I could use your feedback is the README, which I reorganized quite a bit to accommodate the newly added things, i.e. check whether the organization makes sense, is the readme clear, do the instructions work, etc.

File: lungair/README.md [hunk 1]
```diff
@@ -16,67 +17,97 @@ backend Python server for a connected deep learning pipeline.
...
+
+### Electronic health records (EHR)
+
+The LungAIR web application can interact with EHR in a few different ways.
+
+#### Cerner/Oracle test patient login
+
+This proof-of-concept demonstrates user authentication and data retrieval from a Cerner FHIR server.
+
+Go to the _LungAir EHR_ tab in VolView. The login process is currently hard-coded for
```
Comment (jadh4v) [inline lungair/README.md:83]: We don't want to refer to the web application as VolView in the later parts of the ReadMe. I think we can continue to refer to it as _LungAir Web Application_.

File: lungair/README.md [hunk 2]
```diff
@@ -16,67 +17,97 @@ backend Python server for a connected deep learning pipeline.
...
+
+To set up the local FHIR server:
+
+1. Go through the [initial fhir-sandbox setup](https://github.com/KitwareMedical/fhir-sandbox#initial-setup).
+   When setting up a python environment within which to later run the fhir-sandbox scripts, you should additionally
+   `pip install` the following packages: `openpyxl`, `scipy`.
+   We assume that the FHIR server is now running and listening at port 3000.
+2. Edit `lungair/fhir-sandbox-config/lungair_data_source.json` and set `data_file_path` to point to the LungAIR NICU data table.
+   If you do not have this table, then simply set `data_file_path` to be `null` to enable synthetic data generation.
+   This purpose of the synthetic data is only to demo the software interface. With the synthetic data option, you may also
```
Comment (jadh4v) [inline lungair/README.md:104]:
```suggestion
   The purpose of the synthetic data is only to demo the software interface. With the synthetic data option, you may also
```

**Review by jadh4v — APPROVED** [review]
Overall, looking great.

File: lungair/README.md (hunk 1 above)
Comment (ebrahimebrahim) [reply lungair/README.md:83]: Agreed, though let's just call it _LungAIR_ for short where it's clear that it's the web app we are referring to

**Pull request merged**